app/components/chart-options-selector.tsx

```diff
@@ -198,7 +219,86 @@ const EncodingOptionsPanel = ({
   );
 };
 
-const Filter = ({
+const ChartFieldSorting = ({
+  state,
+  field,
+  encodingSortingOptions,
+  disabled = false,
+}: {
+  state: ConfiguratorStateConfiguringChart;
+  field: string;
+  encodingSortingOptions: EncodingSortingOption[];
+  disabled?: boolean;
+}) => {
+  const activeSortingType = get(
+    state,
+    `chartConfig.fields.${field}.sorting.sortingType`,
```

Comment:
Just checking because I remember there was an issue with this in the past: will that not break if `field` contains a `.` for some reason? Probably better use an array as path: `[ "chartConfig", "fields", field, "sorting", "sortingType"]`